common/workunit/workunit.hpp

```diff
@@ -1257,6 +1273,10 @@ interface IConstWorkUnit : extends IConstWorkUnitInfo
     virtual void clearGraphProgress() const = 0;
     virtual IStringVal & getAbortBy(IStringVal & str) const = 0;
     virtual unsigned __int64 getAbortTimeStamp() const = 0;
+    virtual StringBuffer & getSlaveLogFileName(const char *thorProcess, int slaveNum, const char *ipAddress, StringBuffer &logFileName, bool withPath) const = 0;
+    virtual bool usingDedicatedLogFiles(const char* processType, const char* processName, const char* logSpec) const = 0;
+    virtual unsigned getNumberOfThorSlaves(const char * processName) const = 0;
```

Comment:
see other comment, only called in one context, and it is where it is already iterating through processes.
Better to remove.

Reply:
Will remove.